Stream/CloudDataFlowLogSink/Startup.cs

```diff
@@ -0,0 +1,56 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Memory;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+//using Microsoft.OpenApi.Models;
```

Comment:
```suggestion
```

In the interest of keeping samples concise... Are all the other namespaces used?